### PR TITLE
Fixed toggles on main menu

### PR DESCRIPTION
**Pull request by frozenbanana**
Toggles in Settings now are working on Crazy Canvas. 

Overriding in Lobby is still active though.

### CODE REVIEW

File: CrazyCanvas/Source/GUI/MainMenuGUI.cpp [hunk 1]
```diff
@@ -44,7 +44,18 @@ MainMenuGUI::MainMenuGUI(const LambdaEngine::String& xamlFile)
 	FrameworkElement::FindName<TextBox>("CAM_JUMP")	->SetText(KeyToString(InputActionSystem::GetKey("CAM_JUMP")));
 	FrameworkElement::FindName<TextBox>("CAM_SPEED_MODIFIER")->SetText(KeyToString(InputActionSystem::GetKey("CAM_SPEED_MODIFIER")));
 	
+	// Toggles	
 	m_RayTracingEnabled = EngineConfig::GetBoolProperty("RayTracingEnabled");
+	ToggleButton* toggleMeshShader = FrameworkElement::FindName<ToggleButton>("RayTracingToggleButton");
```
Comment (skumbananaaa) [inline CrazyCanvas/Source/GUI/MainMenuGUI.cpp]: Rename to pToggleMeshShaders

File: CrazyCanvas/Source/GUI/MainMenuGUI.cpp [hunk 2]
```diff
@@ -44,7 +44,18 @@ MainMenuGUI::MainMenuGUI(const LambdaEngine::String& xamlFile)
...
+	NS_ASSERT(toggleMeshShader != 0);
+	toggleMeshShader->SetIsChecked(m_RayTracingEnabled);
+	m_RayTracingSleeping = !m_RayTracingEnabled;
+
+	m_MeshShadersEnabled = EngineConfig::GetBoolProperty("MeshShadersEnabled");
+	ToggleButton* toggleRayTracing = FrameworkElement::FindName<ToggleButton>("MeshShadersToggleButton");
```
Comment (skumbananaaa) [inline CrazyCanvas/Source/GUI/MainMenuGUI.cpp]: Rename to pToggleRayTracing

**Review by skumbananaaa — COMMENTED** [review]
Minor code standard changes, otherwise looks good

**Review by TimTheToad — APPROVED** [review]
Looks good, sorry for late Approve/Review!

**Pull request merged**